src/img/lib/cli.js

```diff
@@ -783,11 +783,6 @@ CLI.prototype.do_sources.options = [
         helpArg: '<source>',
         help: 'Add a source. It is appended to the list of sources.'
     },
-    {
-        names: ['add-docker-hub'],
-        type: 'bool',
-        help: 'A shortcut for "imgadm sources -t docker -a https://docker.io".'
-    },
```

Comment:
This doesn't have an sdc-imgapi counterpart?

Reply:
Nope. This is purely for importing images into SmartOS.